hazelcast/src/main/java/com/hazelcast/wan/AddWanConfigResult.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.wan;
+
+import com.hazelcast.config.WanReplicationConfig;
+
+import java.util.Collection;
+
+/**
+ * The result of adding WAN configuration.
+ *
+ * @see WanReplicationService#addWanReplicationConfig(WanReplicationConfig)
+ */
+public class AddWanConfigResult {
+
+    private final Collection<String> addedPublisherIds;
+    private final Collection<String> ignoredPublisherIds;
```

Comment:
can we also have a simple javaDoc to explain what `ignored` and `added` means in this context?

Reply:
Got it, will do.